ee/tabby-db/src/provided_repositories.rs

```diff
@@ -143,4 +145,52 @@ mod tests {
             .await
             .unwrap();
     }
+
+    #[tokio::test]
```

Comment:
i suggest put this in a centralized place like `tabby-db/src/migration_tests/test_0029.rs`